src/Core/Grid/Column/Type/DataColumn.php

```diff
@@ -55,9 +55,11 @@ protected function configureOptions(OptionsResolver $resolver)
             ])
             ->setDefaults([
                 'clickable' => true,
+                'raw' => false,
```

Comment:
I think you can revert this change because of `HtmlColumn`

Reply:
Didn't see that! 😅